sdk/src/main/java/com/sygic/travel/sdk/common/database/di/dbModule.kt

```diff
@@ -7,13 +7,18 @@ import com.github.salomonbrys.kodein.bind
 import com.github.salomonbrys.kodein.instance
 import com.github.salomonbrys.kodein.singleton
 import com.sygic.travel.sdk.common.database.Database
+import com.sygic.travel.sdk.common.database.migrations.migration1_2
```

Comment:
is it ok, that the migration file has dash in it's name, while here is an underscore?

Reply:
Should be, but I will rename it :)